bin/generate.py

```diff
@@ -76,12 +76,12 @@ def assert_type(
     )
 
 
-UNIQUE_TESTCASE_KEYS: Final[Sequence[str]] = [
+UNIQUE_TESTCASE_KEYS: Final[Sequence[str]] = (
     "copy",
     "generate",
     "retries",
     "count",
-] + [e[1:] for e in config.KNOWN_TEXT_DATA_EXTENSIONS]
+) + tuple(e[1:] for e in config.KNOWN_TEXT_DATA_EXTENSIONS)
```

Comment:
To remove the explicit `tuple` functions, this (and similar occurrences) could be written as:

```suggestion
    *(e[1:] for e in config.KNOWN_TEXT_DATA_EXTENSIONS),
)
```

Reply:
not sure if i like this more, but will change.